src/libs/analytics.ts

```diff
@@ -18,6 +18,7 @@ enum AnalyticsEvent {
   CONSULT_WHOLE_OFFER = 'ConsultWholeOffer',
   SHARE_OFFER = 'Share',
   WHY_ANNIVERSARY_CLICKED = 'SignUp-whyAnniversary',
```

Comment:
```suggestion
  WHY_ANNIVERSARY_CLICKED = 'WhyAnniversary',
```

on a pas besoin de rajouter l'information `SignUp`. Quand un event est loggé, le screen est récupéré automatiquement.

Reply:
Faut communiquer cela à l'équipe data

Reply:
Lina me dit que le préfixe est là pour pouvoir avoir une convention de nommage commune : page_évènement

Reply:
commune à quoi ? on fait pas ça sur matomo, au contraire, on distingue bien la catégorie (Home/Offer) de l'action (ConsultOffer/AllModulesSeen).
Et si on utilise une convention comme ça, faudrait voir à quoi correspond `page_`, parce que là dans les 2 evenements que t'as créé, `Signup_whyAnniversary` => ok, page SignUp, mais `Signup_cancelSignup` => on lui passe plusieurs pages ;) en paramètre.

Reply:
L'avantage que je vois à la convention `page_event` c'est pour les funnels, qu'on peut créer directement sur le dashboard firebase, sans passer par BigQuery. Mais de l'autre côté, les noms sont plus longs, et ne sont pas réutilisables sur d'autres pages.

Reply:
en plus, si on adopte une nouvelle convention, il faut mettre à jour tous les anciens évènements :)